Lib/logging/config.py

```diff
@@ -647,6 +647,9 @@ def configure(self):
 
     def configure_formatter(self, config):
         """Configure a formatter from a dictionary."""
+        if isinstance(config, logging.Formatter):
+            return config
+
         if '()' in config:
```

Comment:
Using '()' would be more consistent with handler configuration but then the entry in the dict is just
```python
{
    '()': my_object
}
```

Reply:
Doesn't that work?